modules/options-for-notebook-server-environments.adoc

```diff
@@ -143,19 +226,42 @@ a| * Python 3.8
 * Scikit-learn 0.24
 * SciPy 1.6
 
-| TrustyAI
-| 
+.2+| TrustyAI
+| 2023.2 (Recommended)
 a| * Python 3.9
 * JupyterLab 3.5
```

Comment:
```suggestion
* JupyterLab 3.6
```

Reply:
hmm, looks like this was marked as resolved but never was actually incorporated :/